integration-junit4/src/main/java/com.github.dockerunit.integration/RequestDTO.java

```diff
@@ -0,0 +1,17 @@
+package com.github.dockerunit.integration;
```

Comment:
the file structure is using `.` instead of `/`:
![filepath](https://user-images.githubusercontent.com/630567/107267519-e6f0f600-6a3e-11eb-923a-e4fc738213fc.png)

Reply:
Blame it on IntelliJ :-p

Reply:
@DarthSeppius  :-p